src/DataGrid.tsx

```diff
@@ -151,10 +152,12 @@ export interface DataGridProps<R, SR = unknown, K extends Key = Key> extends Sha
   /**
    * Event props
    */
-  /** Function called whenever a row is clicked */
-  onRowClick?: Maybe<(row: R, column: CalculatedColumn<R, SR>) => void>;
-  /** Function called whenever a row is double clicked */
-  onRowDoubleClick?: Maybe<(row: R, column: CalculatedColumn<R, SR>) => void>;
+  /** Function called whenever a cell is clicked */
+  onCellClick?: RowRendererProps<R, SR>['onCellClick'];
```

Comment:
Can we do the opposite and make `RowRendererProps` get these props from `DataGridProps` instead?